crawl-ref/source/directn.cc

```diff
@@ -557,7 +557,7 @@ class view_desc_proc
     void nextline() { cgotoxy(1, wherey() + 1); }
 };
 
-// Lists all the monsters and items currently in view by the player.
+// Lists monsters, items, and features (including traps) in the player's view.
```

Comment:
"Features" seems a little broad, since every possible tile is a "feature", including floor and wall. It's probably not worth explicitly listing every feature that's included, but maybe just say "and interesting features" or something?

Reply:
Sounds reasonable. I changed it to "some interesting features" since not all interesting features are listed, either; hopefully that strikes you as all right.